src/periphery/MgvReader.sol

```diff
@@ -48,6 +49,23 @@ contract MgvReader {
     bool accumulate;
   }
 
+  /**
+   * @notice Open markets tracking (below) provides information about which markets on Mangrove are open. Anyone can update a market status by calling `updateMarket`. 
+   * @notice The array of pairs `_openMarkets` is the array of all currently open markets (up to a delay in calling `updateMarkets`). A market is a pair of tokens `[tkn0,tkn1]`. The orientation is non-meaningful but canonical (see `order`).
+   * @notice In this contract, 'markets' are defined by non-oriented pairs. Usually markets come with a base/quote orientation. Please keep that in mind.
```

Comment:
I wonder how a UI will determine which token should be considered the base/quote respectively?

Reply:
It should be hardcoded by each UI I think. There are traditions, such as the more cash-like token being the quote. But it's really a matter of convention.